spec/models/user_spec.rb

```diff
@@ -8,6 +8,8 @@
     it { is_expected.to validate_presence_of(:email) }
     it { is_expected.to validate_uniqueness_of(:name) }
     it { is_expected.to validate_uniqueness_of(:email) }
+    it { is_expected.to validate_confirmation_of(:email) }
+    it { is_expected.to validate_length_of(:password) }
```

Comment:
A suggestion to add: `.is_at_least(6).is_at_most(20)`

Reply:
Done in sha: 1159cb8